modules/administration-guide/pages/installing-che-on-minikube-keycloak-oidc.adoc

```diff
@@ -14,9 +14,9 @@ WARNING: Single-node {kubernetes} clusters are suited only for testing or develo
 
 .Prerequisites
 
-* Minikube with {kubernetes} `{kube-ver-min}` or later. See link:https://kubernetes.io/docs/tasks/tools/install-minikube/[Installing Minikube].
+* Minikube stable release. See link:https://kubernetes.io/docs/tasks/tools/install-minikube/[Installing Minikube].
```

Comment:
```suggestion
* Minikube stable release. See the Kubernetes Documentation for link:https://kubernetes.io/docs/tasks/tools/install-minikube/[Installing Minikube].
```

Reply:
For consistency, it doesn't really make sense. We never present external links that way.